### ISSUE TITLE
Is there an option to squash and rebase Hotfix commits on finish?

### ISSUE THREAD

**Issue by simonweil** [petervanderdoes/gitflow#160]
Very much appreciate this project!

**Comment by d1egoaz** [comment]
+1 squash

**Comment by d1egoaz** [comment]
Sorry, this option is already implemented in ```1.8.0 (AVH Edition)```
I just saw [https://github.com/nvie/gitflow/pull/210](https://github.com/nvie/gitflow/pull/210)


Testing:  ```git flow feature finish test -S```
And I got:
```
gf feature finish test -S
Switched to branch 'develop'
Updating b69290f..1bac5a1
Fast-forward
Squash commit -- not updating HEAD

```
:+1:

**Comment by jabbett** [comment]
@d1egoaz I think the request is for `hotfix finish` to have the same `-S` squash option as `feature finish`. I'd very much like this, too.

**Comment by Shumkov** [comment]
:+1:

**Comment by petervanderdoes** [comment]
Implemented in commit 5f2faafc9e0c356a12c55bb363a4dc7230c092c8 Version 1.9.0-dev.53
